sciencebox/values.yaml

```diff
@@ -587,6 +583,120 @@ swan:
     #       cert:
     #       key:
 
+## cbox web
+cernbox-web:
+  initContainers:
+    - name: install-web
+      image: busybox
+      command: ["/bin/sh", "-c"]
```

Comment:
I would actually:
- Move line 593:599 to a bash script
- Make a configMap out of it
- Mount the configMap into the initContainer
- Use it as a command
- Make `https://github.com/cernbox/web-release/releases/download/v0.0.9/web.tar.gz` and `https://github.com/cernbox/web-extensions/releases/download/theme-cernbox/v0.1.7/theme-cernbox.tar.gz` ENV_VARs that the script uses